packages/dev/core/src/Materials/Node/nodeMaterial.ts

```diff
@@ -2153,6 +2156,65 @@ export class NodeMaterial extends PushMaterial {
         this._mode = NodeMaterialModes.Particle;
     }
 
+    /**
+     * Clear the current material and set it to a default state for gaussian splatting
+     */
+    public setToDefaultGaussianSplatting() {

```

Comment:
My only ask here would be to make that an external function so we do not grow the size of nodeMaterial whennot using GS. I should have done that for other modes but it is too late now

Reply:
I've moved the function into `nodeMaterialDefault.ts`

Reply:
Merging now.